test-resources/odata-service-specs/v4/API_TEST_SRV/API_TEST_SRV.edmx

```diff
@@ -224,6 +239,10 @@
           <NavigationPropertyBinding Path="to_OtherMultiLink" Target="A_TestEntityMultiLink"/>
           <NavigationPropertyBinding Path="to_SingleLink" Target="A_TestEntitySingleLink"/>
         </EntitySet>
+        <EntitySet Name="A_TestEntitySharesEntityType1" EntityType="API_TEST_SRV.A_TestEntityTypeToBeShared" sap:content-version="1">
+        </EntitySet>
+        <EntitySet Name="A_TestEntitySharesEntityType2" EntityType="API_TEST_SRV.A_TestEntityTypeToBeShared" sap:content-version="1">
+        </EntitySet>

```

Comment:
These two entity sets share the same entity type `API_TEST_SRV.A_TestEntityTypeToBeShared`